### PR TITLE
Update to level 1 fit function

### PR DESCRIPTION
**Pull request by andrew-aws**
Moved jet_synchtest to jet_match, and changed some data handling internally to improve efficiency. Added new fitting functions, as well as sanity check plots to jet_syncfit. Fixed some bugs in SynchFittingProcedure, Style.h, and HistogramUtilities.

### CODE REVIEW

**Review by ktht — CHANGES_REQUESTED** [review]
Since you renamed `JetAnalyzers/bin/jet_synchtest_x.cc` to `JetAnalyzers/bin/jet_match_x.cc`, I'd also suggest to change `BuildFile.xml` accordingly.

**Comment by andrew-aws** [comment]
Just added the BuildFile as well as classes_def files

File: JetAnalyzers/bin/BuildFile.xml [hunk 1]
```diff
@@ -40,6 +40,7 @@
 <bin   name="jet_draw_pileup_comparison_x"	   file="jet_draw_pileup_comparison_x.cc" />
 <bin   name="jet_copyFiles_x"                  file="jet_copyFiles_x.cc" />
 <bin   name="jet_synchtest_x"                  file="jet_synchtest_x.cc" />
```
Comment (ktht) [inline JetAnalyzers/bin/BuildFile.xml]: Line containing `jet_synchtest_x` should be removed.

File: JetUtilities/src/classes_def.xml [hunk 2]
```diff
@@ -49,6 +49,7 @@
   <class name="std::vector<std::pair<ULong64_t,ULong64_t> >"/>
   <class name="edm::Wrapper<std::vector<std::pair<ULong64_t,ULong64_t> > >"/>
   <class name="TProfileMDF" ClassVersion="1">
+   <version ClassVersion="1" checksum="538640849"/>
```
Comment (ktht) [inline JetUtilities/src/classes_def.xml:52]: This works only in CMSSW 10X but breaks in CMSSW 9X area.

**Pull request closed without merging**